parlai/agents/memnn/memnn.py

```diff
@@ -182,20 +187,20 @@ def _build_mems(self, mems):
         padded = torch.LongTensor(bsz, num_mems, seqlen).fill_(0)
 
         for i, mem in enumerate(mems):
-            # tf_offset = len(mem) - 1
+            tf_offset = len(mem) - 1
```

Comment:
weird that this was commented before?

Reply:
yeah, why were these commented out?

Reply:
Okay I'm still confused why these lines were commented out before.